src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -45,6 +45,15 @@
 					@click="showFilePicker = true">
 					{{ t('spreed', 'Browse …') }}
 				</NcButton>
+				<FilePickerVue v-if="showFilePicker"
+					:name="t('spreed', 'Select location for attachments')"
+					:path="attachmentFolder"
+					container=".app-settings-section__wrapper"
+					:buttons="filePickerButtons"
+					:multiselect="false"
+					:mimetype-filter="['httpd/unix-directory']"
+					allow-pick-directory
+					@close="showFilePicker = false" />
```

Comment:
I don't think there is a difference in putting here or in its previous position in DOM

Reply:
It will be mounted in specified class selector anyway, but I think it would be better to keep them closer with other logic